README.md

```diff
@@ -38,14 +38,14 @@ limitations under the License.
 </p>
 
 <h3 align="center">
-<p>State-of-the-art Natural Language Processing for PyTorch and TensorFlow 2.0
+<p>State-of-the-art Natural Language Processing for Jax, PyTorch and TensorFlow
```

Comment:
Should we maybe say `Flax/Jax` instead to make it clear that we only support Jax in combination with Flax?

Reply:
That's too long for the subtitle IMO.

Reply:
Just specify it in the `installation` part.

Reply:
Think I'd rather than say **Flax** instead of **Jax** since Jax could also mean that we support the Trax library or Haiku library -> What do you think?

Reply:
Ok just read your argument about Jax being more popular -> that's a good point. Ok for me then!